src/Symfony/Component/HttpKernel/Profiler/Profiler.php

```diff
@@ -34,8 +34,8 @@ class Profiler implements ResetInterface
     private array $collectors = [];
 
     private ?LoggerInterface $logger;
-    private bool $initiallyEnabled = true;
-    private bool $enabled = true;
```

Comment:
Good catch. But is it a breaking change if someone extends this class without calling the parent constructor? @stof @nicolas-grekas

Reply:
Let's revert this as it's not needed, right?

Reply:
It's not needed, but it seemed redundant.